packages/forma-36-website/src/content/integrations/react-hook-form/index.mdx

```diff
@@ -81,15 +80,15 @@ export const ReactHookForm = () => {
       <FormControl>
         <FormControl.Label>Martial status</FormControl.Label>
 
-        {/* When using RadioGroup, it’s necessary to register each Radio with the same "name" */}
-        <RadioGroup defaultValue="single">
+        {/* When using Radio.Group, it’s necessary to register each Radio with the same "name" */}
```

Comment:
Can we adjust the comment and say, that it's the react hook that need's that? One of the comments was around the fact that you need to register on the input level, not on the group level. I imagine, that if you have many inputs in the group it might be annoying to register each of them. So maybe let's make it clear that this is what this hook requires, and if the user doesn't want to do that, they can use formik. WDYT?